### ISSUE TITLE
no auto rotate back with xbox controller

### ISSUE THREAD

**Issue by olliebob** [Mogli12/KeyboardSteer#115]

**Comment by Mogli12** [comment]
Does it work without KeyboardSteer? Because the script does not change auto rotate back speed anymore besides the action binding ksmNO_ARB.

**Comment by olliebob** [comment]
Sorry what I’m wanting is for when I turn with the controller I want the wheels to stay in that direction not return to centre. Yours has a feature (no auto rotate bk) but with a controller it still rotates to centre. Drive control in 17 had a feature (real steer) that worked with a controller and I wondered if u could implement it. Everything else works as should, brilliant I’ve implemented a gearbox (major gearbox mod fan and MR fan)

Sent from my iPhone

On 15 Jan 2019, at 05:54, Stefan Biedenstein <notifications@github.com<mailto:notifications@github.com>> wrote:


Does it work without KeyboardSteer? Because the script does not change auto rotate back speed anymore besides the action binding ksmNO_ARB.

**Comment by Mogli12** [comment]
You can reduce the auto rotate back speed in the normal game settings w/o keyboard steer.

**Comment by olliebob** [comment]
Yes I know but unfortunately it doesn’t have any effect with a controller, it a huge shame

Sent from my iPhone

On 15 Jan 2019, at 07:46, Stefan Biedenstein <notifications@github.com<mailto:notifications@github.com>> wrote:


You can reduce the auto rotate back speed in the normal game settings w/o keyboard steer.

**Comment by cabmac** [comment]
you can set non auto rotate in the settings of the game...

**Comment by olliebob** [comment]
Yes I know but it doesn’t work with controllers, yet again a simple addon from giants but they couldn’t get that right

Sent from my iPhone

On 17 Jan 2019, at 11:32, cabmac <notifications@github.com<mailto:notifications@github.com>> wrote:


you can set non auto rotate in the settings of the game...

**Comment by Mogli12** [comment]
I think auto rotate back is only for digital controllers. But I assume that your controller has an analog axis and in this case the steering angle corresponds directly to the value of the axis. This is NOT auto rotate back.

But this has nothing to do with KeyboardSteer and I do not plan to change it.

**Comment by olliebob** [comment]
Yes it is such a shame as even with an Xbox controller steering is very jerky and with drive control in 17  that implemented it to overcome that and it was excellent. Ets2 have added it it in the latest patches and is excellent with a controller now

Sent from my iPhone

On 17 Jan 2019, at 15:22, Stefan Biedenstein <notifications@github.com<mailto:notifications@github.com>> wrote:


I think auto rotate back is only for digital controllers. But I assume that your controller has an analog axis and in this case the steering angle corresponds directly to the value of the axis. This is NOT auto rotate back.

But this has nothing to do with KeyboardSteer and I do not plan to change it.
